### DISCUSSION TITLE
Using Breakpoints on something like Modal Size

### DISCUSSION THREAD

**Discussion by dl-web-dev** [heroui-inc/heroui#1417] [Help] [answered]
I was curious if its possible to use breakpoints / media queries for something like the Size prop on the Modal Component in v2.

For Ex if I wanted the modal to be fullscreen on mobile but on desktop use the regular size. 

Would something like this be currently possible?

**Comment by jrgarciadev** [comment, marked as answer]
Hey @dml0031 it is possible, you only need to pass the max-width class to the Modal

example:

```tsx
<Modal className="max-w-full h-[100dvh] rounded-none md:max-w-md">
...
</Modal>

```


Codesandbox: https://codesandbox.io/p/sandbox/thirsty-frog-hrcw8l?file=/App.jsx:7,11&utm_medium=sandpack

**Reply by dl-web-dev** [reply]
I tried to use just widths and ultimately didn't work well, at least when trying to switch between full screen and regular sizing on desktop. 

I believe it has to do with how the scrollableType plays as well. I need the scrollable type to be inside however it doesn't really appear to support a full screen modal with inside scrolling
